src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.MaxConnectionsPerServer.cs

```diff
@@ -30,6 +30,7 @@ public void Default_ExpectedValue()
         [Theory]
         [InlineData(0)]
         [InlineData(-1)]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/28882", TargetFrameworkMonikers.NetFramework)]
```

Comment:
nit:  I thought we all agreed to use just numbers for the ActiveIssue and not these long URL's, especially for issues in the same repo as the tests being disabled.

cc: @danmosemsft

Reply:
I wasn't aware of that. I thought that for the networking tests you guys liked the long paths. Personally I like just the numbers, will update it to include the numbers only.

Reply:
I remember a discussion about it, don't remember the resolution, but I like numbers also

I think we just de facto do what the library owner prefers..

Reply:
I don't know if I qualify as a "library owner" but I prefer simpler things like numbers that are easier to maintain in the code.